### PR TITLE
Fix manual docker build

### PR DESCRIPTION
**Pull request by mmochan**
#### What does this PR do?
Fixes the Github action for manual build

#### Type of change
- [x] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: .github/workflows/docker-latest.yml [hunk 1]
```diff
@@ -3,8 +3,7 @@ name: docker-latest
 on: [workflow_dispatch]
 
 env:
-  ORG: opendatacube
-  IMAGE: wps
+  IMAGE_NAME: opendatacube/wps
```
Comment (pindge) [inline .github/workflows/docker-latest.yml:6]: i wonder if the name should be `datacube-wps` like how explorer and ows are named

**Review by pindge — DISMISSED** [review]

File: .github/workflows/docker-latest.yml (hunk 1 above)
Comment (mmochan) [reply .github/workflows/docker-latest.yml:6]: The Github action scan.yml file refers to 

`IMAGE_NAME: opendatacube/wps`

Comment (uchchwhash) [reply .github/workflows/docker-latest.yml:6]: I would agree with @pindge , we should probably change this to `datacube-wps` everywhere.

Comment (uchchwhash) [reply .github/workflows/docker-latest.yml:6]: but this PR can go on without it 👍🏽

**Review by uchchwhash — APPROVED** [review]

**Pull request merged**